src/phoenix/db/migrations/versions/bc8fea3c2bc8_add_prompt_tables.py

```diff
@@ -149,7 +149,7 @@ def upgrade() -> None:
     )
 
     op.create_table(
-        "prompt_template_version_tags",
+        "prompt_tags",
```

Comment:
These are explicitly tags on versions though. I think you should leave version in the name